src/presentation/test/integration/registration/limitedPartner/add-limited-partner-person.test.ts

```diff
@@ -26,7 +29,58 @@ describe("Add Limited Partner Person Page", () => {
   });
 
   describe("Get Add Limited Partner Page", () => {
-    it("should load the add limited partner page with Welsh text", async () => {
+    it("should load the add limited partner page with Welsh text for LP", async () => {
```

Comment:
could you use a it.each instead, you could pass the partnership type and a boolean as params into the test, the boolean could indicate whether capitalContribution should be present or not. it should reduce the amount of test code needed.

Reply:
Updated to use it.each with parameters.